defaults/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+mariadb_version: 10.1
+mysql_port: 3306
+mysql_socket: "/var/lib/mysql/mysql.sock"
+mysql_bind_address: "{{ ansible_eth0.ipv4.address }}"
```

Comment:
Use {{ frontnet_addr }} for now if we're doing this on the frontnet.  That allows the caller to determine what nic it will be mapped to.